datafusion/physical-expr/src/datetime_expressions.rs

```diff
@@ -17,6 +17,8 @@
 
 //! DateTime expressions
 
+use crate::datetime_expressions;
```

Comment:
```suggestion
```
It seems that this `use` may not be required 🤔.

Reply:
🤔  but clippy did not flag it